src/router/definitions.ts

```diff
@@ -19,6 +19,7 @@ export interface ProjectsParams {
     | { resource: "patch"; params: { patch: string } }
     | { resource: "patches" };
   hash?: string;
```

Comment:
Remove?

Reply:
I kept the `hash` property, so we can allow people that come to the web client with a hash param in their URL visiting a markdown document to handle that.
https://github.com/radicle-dev/radicle-interface/blob/f8ddbbf5daf5195c556218d591a1a5a64769175c/src/base/projects/Readme.svelte#L11

Reply:
Ah ok.